src/reference/antora/modules/ROOT/pages/ip/tcp-connection-factories.adoc

```diff
@@ -194,6 +194,26 @@ As noted there, such modifications are possible if SSL is being used, or not.
 
 Also see xref:ip/annotation.adoc[Annotation-Based Configuration] and xref:ip/dsl.adoc[Using the Java DSL for TCP Components].
 
+[[host-verification]]
+== Host verification
+Starting from version 5.1.0, host verification is enabled by default for enhanced security. This feature ensures that the server's identity is verified during TCP connections.
+
+If you encounter a scenario where host verification needs to be disabled (not recommended), you can configure the socket-support attribute in the tcp-connection-factory.
+
+[source,xml]
+----
+<int-ip:tcp-connection-factory id="client"
+                                type="client"
+                                host="localhost"
+                                port="0"
+                                socket-support="customSocketSupport"
+                                single-use="true"
+                                so-timeout="10000"/>
+<bean id="customSocketSupport" class="org.springframework.integration.ip.tcp.connection.DefaultTcpSocketSupport">
```

Comment:
Better to have a blank line between components definitions: much easier to read the code.
Please, also ensure that not tab indents in this code snippet.
Yes, we prefer tabs in the code base, but in the docs it is somehow better with spaces.
Sorry for inconvenience.

Reply:
please check. Should be resolved.